src/Utils/apolloClient.ts

```diff
@@ -1,7 +1,7 @@
 import { ApolloClient, InMemoryCache } from "@apollo/client";
 
 const client = new ApolloClient({
-  uri: "https://jabref-online.herokuapp.com",
+  uri: process.env.API_HOST,
```

Comment:
I think it would be good to still fall-back to the heroku url in case the developer doesn't provide something in the env file.